### ISSUE TITLE
Aria labels for table actions need improvement

### ISSUE THREAD

**Issue by Macroz** [CSCfi/rems#1655]
- [ ] Aria label for view command in continue application table ~~does not contain a useful title of the application.~~ The label should be improved.
- [ ] Aria label for view command in applications table ~~does not contain a useful title of the application or the resources applied but only the state.~~ The label should be improved.

**Comment by luontola** [comment]
The view button in applications table already contains the application's description.

**Comment by luontola** [comment]
The view button in the continue application table is the same component, but new applications don't have a description so it's usually empty.

**Comment by luontola** [comment]
What kind of an improvement does it need, then?

**Comment by Macroz** [comment]
e.g. in the continue application table, if there is no description, the names of the resources can be read instead. Same applies to the view action in the applications table. Something else can be read instead, perhaps even the state.
